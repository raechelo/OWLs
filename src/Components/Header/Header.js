import React from 'react';

function Header() {

  return (
    <div className="header-img">
      <img src="https://fontmeme.com/permalink/190412/3229c92253ff3dd3d93ac8c7f060851e.png" alt="Header logo" />
    </div>
  )
}


export default Header;