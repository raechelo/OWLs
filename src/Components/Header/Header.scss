.header-img {
  text-align: center;
  background-color: $boardBg;
  margin: 3rem 4rem;
  width: 50%;
}