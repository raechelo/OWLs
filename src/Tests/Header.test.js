import React from 'react';
import Header from '../Components/Header/Header';
import { shallow } from 'enzyme';

describe('Header', () => {

  let wrapper;

  beforeEach(() => {
    wrapper = shallow (
      <Header />
    )
  });

  it('should match the snapshot with all the data passed in', () => {
    expect(wrapper).toMatchSnapshot()
  });
})

