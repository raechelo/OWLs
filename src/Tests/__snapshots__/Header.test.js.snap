// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header should match the snapshot with all the data passed in 1`] = `
<div
  className="header-img"
>
  <img
    alt="Ordinary [React] Wizarding Level"
    src="https://fontmeme.com/permalink/190412/3229c92253ff3dd3d93ac8c7f060851e.png"
  />
</div>
`;
